flask_caching/__init__.py

```diff
@@ -180,6 +182,7 @@ def init_app(self, app, config=None):
         config.setdefault("CACHE_ARGS", [])
         config.setdefault("CACHE_TYPE", "null")
         config.setdefault("CACHE_NO_NULL_WARNING", False)
+        config.setdefault("CACHE_SOURCE_CHECK", False)
```

Comment:
Please add this new config variable to th edocumentation.

Reply:
Added it to docs/index.rst. Please have a look and let me know if it looks good.